hw_lesson 14.py

```diff
@@ -0,0 +1,132 @@
+
+first_channel() - turns on the first channel from the list.
+last_channel() - turns on the last channel from the list.
+turn_channel(N) - turns on the N channel. Pay attention that the channel numbers start from 1, not from 0.
+next_channel() - turns on the next channel. If the current channel is the last one, turns on the first channel.
+previous_channel() - turns on the previous channel. If the current channel is the first one, turns on the last channel.
+current_channel() - returns the name of the current channel.
+is_exist(N/'name') - gets 1 argument - the number N or the string 'name' and returns "Yes",
+if the channel N or 'name' exists in the list, or "No" - in the other case.
+ 
+The default channel turned on before all commands is №1.
+
+Your task is to create the TVController class and methods described above.
+"""
+
+
+class TVController():
+
+    now_current_channel = ''
+
+    def __init__(self, channels: list):
+        self.channels = channels
+        TVController.now_current_channel = self.channels[0]
+
+    def first_channel(self):
+        TVController.now_current_channel = self.channels[0]
+        print(self.channels[0])
+
+    def last_channel(self):
+        TVController.now_current_channel = self.channels[-1]
+        print(self.channels[-1])
+
+    def turn_channel(self, i):
+        TVController.now_current_channel = self.channels[i-1]
+        print(self.channels[i-1])
+
+    def next_channel(self):
+        if TVController.now_current_channel == self.channels[-1]:
+            TVController.now_current_channel = self.channels[0]
+            print(self.channels[0])
+        else:
+            cur_ind = self.channels.index(TVController.now_current_channel)
+            next_ind = cur_ind + 1
+            TVController.now_current_channel = self.channels[next_ind]
+            print(self.channels[next_ind])
+
+    def previous_channel(self):
+        if TVController.now_current_channel == self.channels[0]:
+            TVController.now_current_channel = self.channels[-1]
+            print(self.channels[-1])
+        else:
+            cur_ind = self.channels.index(TVController.now_current_channel)
+            next_ind = cur_ind - 1
+            TVController.now_current_channel = self.channels[next_ind]
+            print(self.channels[next_ind])
+
+    def current_channel(self):
+        print(TVController.now_current_channel)
+
+    def is_exist(self, input_value):
+        if isinstance(input_value, int):
+            if input_value <= len(self.channels) + 1:
+                print('Yes')
+            else:
+                print('No')
+        elif isinstance(input_value, str):
+            if input_value in self.channels:
+                print('Yes')
+            else:
+                print('No')
+        else:
+            print('Unsupported input')
+
+CHANNELS = ["BBC", "Discovery", "TV1000"]
+
+controller = TVController(CHANNELS)
+
+controller.first_channel()
```

Comment:
Please, use this test cases.
```suggestion
assert controller.first_channel() == "BBC"
assert controller.next_channel() == "Discovery"
assert controller.previous_channel() == "BBC"
assert controller.current_channel() == "BBC"
assert controller.is_exist(4) == "No"
assert controller.is_exist("BBC") == "Yes"
```

Reply:
Done
changed code https://github.com/DesoPy/Beetroot_Home_Work/blob/Lesson14/hw_lesson%2014.py